### PR TITLE
Prepared plots for themes

### PR DESCRIPTION
**Pull request by raviselker**

### CODE REVIEW

File: R/descriptives.b.R [hunk 1]
```diff
@@ -123,21 +123,20 @@ descriptivesClass <- R6::R6Class(
...
                 geom_bar(stat="identity") +
                 labs(list(x=NULL, y=ylab)) +
-                theme(
-                    text=element_text(size=16, colour='#333333'),
-                    plot.background=element_rect(fill='transparent', color=NA),
-                    panel.background=element_rect(fill='#E8E8E8'),
-                    axis.title.y=element_text(margin=margin(0,10,0,0)),
-                    axis.text.y=axis.text.y)
+                theme +
+                theme(axis.text.y=axis.text.y,
```
Comment (jonathon-love) [inline R/descriptives.b.R:138]: can and does this work? i can't see how it can. how does it know which theme is which?

Comment (raviselker) [reply R/descriptives.b.R:138]: Why wouldn't it work?

Comment (jonathon-love) [reply R/descriptives.b.R:138]: there two things called `theme`, one is a theme object, the other is a function. how does it know which is which?

Comment (raviselker) [reply R/descriptives.b.R:138]: they exist in different environments I guess

Comment (jonathon-love) [reply R/descriptives.b.R:138]: so i guess you've tried it, and it works, but i'm still pretty surprised.

Comment (raviselker) [reply R/descriptives.b.R:138]: Yup, it does work

**Pull request merged**